frontend/ui/game.tsx

```diff
@@ -255,14 +259,26 @@ export class Game extends React.Component {
     if (!this.state.settings.fullscreen) {
       shareLink = (
         <div id="share">
-          Send this link to friends:
+          Send this link to friends:&nbsp;
           <a className="url" href={window.location.href}>
             {window.location.href}
           </a>
         </div>
       );
     }
 
+    const timer = !!this.state.game.timer_settings?.length && (
+      <div id="timer">
+        <Timer
+          endTime={new Date(this.state.game.end_time).getTime() + 1000}
+          handleExpiration={() => {
+            this.endTurn();
+          }}
```

Comment:
https://github.com/jbowens/codenames/issues/70#issuecomment-611253688

In this issue, I thought we shouldn't add automatic turn changing unless it's configurable. How about we omit this bit for now. When the timer hits zero, it should just stop. Then as a followup we can add a toggle somewhere to support automatically ending the turn.